pigar/parser.py

```diff
@@ -59,9 +64,16 @@ def _read_code(fpath):
     if fpath.endswith(".ipynb"):
         nb = nbformat.read(fpath, as_version=4)
         code = ""
-        for cell in nb.cells:
-            if cell.cell_type == "code":
-                code += cell.source + "\n"
+        if PYTHON_VERSION_3:
+            transformer = IPython.core.inputtransformer2.TransformerManager()
+            for cell in nb.cells:
+                if cell.cell_type == "code":
+                    code += transformer.transform_cell(cell.source) + "\n"
```

Comment:
Tests failed maybe because the `transform_cell` transformed the original code, such as adding empty lines.